packages/editor/src/store/actions.js

```diff
@@ -44,15 +46,20 @@ export function resetPost( post ) {
  * Returns an action object used in signalling that the latest autosave of the
  * post has been received, by initialization or autosave.
  *
- * @param {Object} post Autosave post object.
+ * @param {Object} newAutosave Autosave post object.
```

Comment:
nit: Should we add a `@deprecated` comment here?

Reply:
Definitely. Done.